src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -38,6 +37,14 @@
                     Maintainer
                   %i.fas.fa-spinner.fa-spin.d-none
 
+  - if write_access && feature_enabled?(:responsive_ux)
```

Comment:
I would simplify this view by removing the `feature_enabled?` check and remove the other block on line 3. The `render partial: ...` call on line 9 would then be moved under this block on line 40.

Reply:
Okay!